rust/lance-encoding/src/decoder.rs

```diff
@@ -750,11 +756,31 @@ impl CoreFieldDecoderStrategy {
                 column_info.as_ref(),
                 self.decompressor_strategy.as_ref(),
             )?);
+
+            // advance to the next top level column
             column_infos.next_top_level();
+
             return Ok(scheduler);
         }
         match &data_type {
             DataType::Struct(fields) => {
+                let field_metadata = &field.metadata;
+                if field_metadata
+                    .get("packed")
+                    .map(|v| v == "true")
```

Comment:
Let's make this case insensitive.

Also, this check repeats in a few places.  We could maybe add `is_packed_struct` to `FieldExt` in `lance-arrow/src/schema.rs`

Reply:
thanks, fixed.
I added two `is_packed_struct`, one in `lance-arrow/src/schema.rs`, one in `lance-core/src/datatypes.field.rs`, not sure how to do only one.